packages/components/src/dimension-control/types.ts

```diff
@@ -14,7 +15,10 @@ export type Size = {
 	slug: string;
 };
 
-export type DimensionControlProps = {
+export type DimensionControlProps = Pick<
+	SelectControlProps,
+	'__next40pxDefaultSize' | '__nextHasNoMarginBottom'
+> & {
```

Comment:
Not picking other props like `value` or `label` because they either have different descriptions or different `required` settings.